tests/positive/Isabelle/Package.juvix

```diff
@@ -0,0 +1,5 @@
+module Package;
+
+import PackageDescription.V2 open;
+
+package : Package := defaultPackage {name := "isabelle"};
```

Comment:
```suggestion
package : Package :=
  defaultPackage@?{
    name := "isabelle"
  };
```

We should use the new named application syntax now we've merged https://github.com/anoma/juvix/pull/2876